data/json/npcs/isolated_road/isolated_road_cody_dialogue.json

```diff
@@ -57,7 +57,7 @@
         "topic": "TALK_BLACKSMITH_PROTOTYPE"
       },
       {
-        "text": "[Show Cody the nomad jumpsuit blueprint.] Hey - I met a group with some interesting ideas for armor design.  Is any of this of use to you?",
+        "text": "[Show the nomad jumpsuit blueprint.] Hey - I met a group with some interesting ideas for armor design.  Is any of this of use to you?",
```

Comment:
Full stop before `]`?

Reply:
I don't follow - do you mean to have two spaces between `]` and `Hey`?

Reply:
I think they mean you shouldn't have the period before the closing bracket.

Reply:
Ah, that's intentional. It's a full sentence describing what the PC is doing, so it has a period.

Reply:
I'm stupid too, so the language and I are a good fit. :p